Healthy/Classes/Modules/SavedRecipes/ MyTableViewCell.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+class MyTableViewCell: UITableViewCell {
+    // swiftlint:disable all
+    private let myImageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let subtitle1Label = UILabel()
+    private let subtitle2Label = UILabel()
+    private let subtitle3Label = UILabel()
```

Comment:
Lastly, this can be renamed to `cookingTimeInMinutesLabel` or `preparationTimeInMinutesLabel`